lib/cocoapods/target/framework_paths.rb

```diff
@@ -34,7 +34,7 @@ def hash
       end
 
       def all_paths
-        [source_path, dsym_path].compact
+        [source_path, dsym_path, bcsymbolmap_paths].compact
```

Comment:
@amorde do you think we want this? It makes perfect sense for this class but this propagates to input paths of script phases...As I am typing this I think we need it and that was a bug in the original PR.

Reply:
yes definitely, I had thought this was covered in the review but I guess it was just included in `hash` https://github.com/CocoaPods/CocoaPods/pull/8470#discussion_r252100005

Reply:
could you add a test for this? just realized there aren't any

Reply:
yes sir!

Reply:
added